dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -6561,6 +6562,42 @@ bool isForeignTableUpdate(THD* thd)
     return false;
 }
 
+bool isMCSTableUpdate(THD* thd)
+{
+    LEX* lex = thd->lex;
+
+    if ((lex->sql_command != SQLCOM_UPDATE_MULTI) &&
```

Comment:
What if we use isUpdateStatement() and isDeleteStatement here and in  isMCSTableDelete()?

Reply:
Done.